server/middlewares/email-not-verified-notice.js

```diff
@@ -0,0 +1,33 @@
+import dedent from 'dedent';
+
+const ALLOWED_METHODS = ['GET'];
+const EXCLUDED_PATHS = [
+  '/api/flyers/findOne',
+  '/signout',
+  '/update-email',
+  '/passwordless-change'
```

Comment:
Please add `/external/services/user` to the excluded paths, else Learn cannot redirect unverified users correctly